ext/opentelemetry-ext-django/src/opentelemetry/ext/django/__init__.py

```diff
@@ -0,0 +1,72 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+
+from django.conf import settings
+
+from opentelemetry.auto_instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.configuration import Configuration
+from opentelemetry.ext.django.middleware import DjangoMiddleware
+
+_logger = getLogger(__name__)
+
+
+class DjangoInstrumentor(BaseInstrumentor):
+    """An instrumentor for Django
+
+    See `BaseInstrumentor`
+    """
+
+    _opentelemetry_middleware = ".".join(
+        [DjangoMiddleware.__module__, DjangoMiddleware.__qualname__]
+    )
+
+    def _instrument(self, **kwargs):
+
+        # FIXME this is probably a pattern that will show up in the rest of the
+        # ext. Find a better way of implementing this.
+        # FIXME Probably the evaluation of strings into boolean values can be
+        # built inside the Configuration class itself with the magic method
+        # __bool__
+
+        if Configuration().DJANGO_INSTRUMENT != "True":
```

Comment:
We should be explicit in telling users on one consistent way of instrumenting their django apps. A user could very easily not use the `Instrumenter` and simply add the path to the middleware in `settings.py` for their application. The importance of using `instrumenter`, AND setting `DJANGO_INSTRUMENT ` must be emphasized, either in text in the examples, in a configuration section in the README and/or the docstrings in this file.

Reply:
Fixed